backend/db/structure.sql

```diff
@@ -152,8 +152,8 @@ ALTER SEQUENCE public.active_storage_variant_records_id_seq OWNED BY public.acti
 CREATE TABLE public.admins (
     id bigint NOT NULL,
     user_id uuid NOT NULL,
-    created_at timestamp with time zone NOT NULL,
-    updated_at timestamp with time zone NOT NULL
+    created_at timestamp(6) with time zone NOT NULL,
+    updated_at timestamp(6) with time zone NOT NULL
```

Comment:
we had this occur before,  as I recall it's caused by rails defaults for timestamp.  maybe it's a side effect of the rails 7 migration?